### PR TITLE
Update org.eclipse.osgi to 3.18.600

### PR DESCRIPTION
**Pull request by tjwatson**

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

File: dev/org.eclipse.osgi/bnd.bnd [hunk 1]
```diff
@@ -10,14 +10,14 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
--include= jar:${fileuri;${repo;org.eclipse.platform:org.eclipse.osgi;3.18.300}}!/META-INF/MANIFEST.MF,bnd.overrides
+-include= jar:${fileuri;${repo;org.eclipse.platform:org.eclipse.osgi;3.18.600}}!/META-INF/MANIFEST.MF,bnd.overrides
```
Comment (dazavala) [inline dev/org.eclipse.osgi/bnd.bnd:13]: Update copyright.

**Review by dazavala — DISMISSED** [review]
Approved in case you opt not to push the copyright update.

**Comment by tjwatson** [comment]
#libby

**Review by dazavala — APPROVED** [review]

**Pull request merged**